### ISSUE TITLE
Adding Client Token signing algorithms results in Error

### ISSUE THREAD

**Issue by IvoryWolf** [skoruba/IdentityServer4.Admin#832]
### Describe the bug
An error page is shown when trying to add additional token signing algorithms to a Client.
The selected items are not saved when reloading the client page. 

### To Reproduce

1. Create a client. 
2. Add any item from from the suggested items. I used ES256, ES384, ES512.
3. Save the client.

![afbeelding](https://user-images.githubusercontent.com/5112678/116084143-c0d20e00-a69d-11eb-82d9-284d5dcf8941.png)

### Relevant parts of the log file
```
{
  "TimeStamp": "2021-04-26T10:57:29.2887473",
  "Level": "Error",
  "Message": "Exception at route /Configuration/Client/3",
  "MessageTemplate": "Exception at route /Configuration/Client/3",
  "Exception": "System.NullReferenceException: Object reference not set to an instance of an object.\r\n   at Skoruba.IdentityServer4.Admin.BusinessLogic.Helpers.ComboBoxHelpers.PopulateValuesToList(String jsonValues, List`1 list)\r\n   at Skoruba.IdentityServer4.Admin.BusinessLogic.Services.ClientService.PopulateClientRelations(ClientDto client)\r\n   at Skoruba.IdentityServer4.Admin.BusinessLogic.Services.ClientService.BuildClientViewModel(ClientDto client)\r\n   at Skoruba.IdentityServer4.Admin.UI.Areas.AdminUI.Controllers.ConfigurationController.Client(ClientDto client)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextExceptionFilterAsync>g__Awaited|25_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ExceptionContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeNextResourceFilter()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Authorization.Policy.AuthorizationMiddlewareResultHandler.HandleAsync(RequestDelegate next, HttpContext context, AuthorizationPolicy policy, PolicyAuthorizationResult authorizeResult)\r\n   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Localization.RequestLocalizationMiddleware.Invoke(HttpContext context)\r\n   at NWebsec.AspNetCore.Middleware.Middleware.CspMiddleware.Invoke(HttpContext context)\r\n   at NWebsec.AspNetCore.Middleware.Middleware.MiddlewareBase.Invoke(HttpContext context)\r\n   at NWebsec.AspNetCore.Middleware.Middleware.MiddlewareBase.Invoke(HttpContext context)\r\n   at NWebsec.AspNetCore.Middleware.Middleware.MiddlewareBase.Invoke(HttpContext context)\r\n   at NWebsec.AspNetCore.Middleware.Middleware.MiddlewareBase.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)",
  "Properties": {
    "SourceContext": "Skoruba.IdentityServer4.Admin.UI.Areas.AdminUI.Controllers.ConfigurationController",
    "ActionId": "fbd4581e-d59a-45b5-896f-49f7d001250d",
    "ActionName": "Skoruba.IdentityServer4.Admin.UI.Areas.AdminUI.Controllers.HomeController.Error (Skoruba.IdentityServer4.Admin.UI)",
    "RequestId": "8000005e-0002-f700-b63f-84710c7967bb",
    "RequestPath": "/Configuration/Client/3",
    "ApplicationName": "IdentityProvider.Admin"
  }
}
```

**Comment by skoruba** [comment]
Thanks! I will fix it.

**Comment by skoruba** [comment]
Fixed in 2.0.1, please check it, thanks 👍

**Comment by IvoryWolf** [comment]
Can confirm everything is working in 2.0.1. 
Thanks for the quick response! 👍
